### PR TITLE
Hide comments in fewer clicks

### LINKED ISSUE #1268: Add shortcut for hiding a comment as "Resolved"
[GitHub introduced support for hiding comments](https://blog.github.com/2018-04-18-new-tools-for-open-source-maintainers/), but it requires many clicks to hide a comment as "Resolved". Since that's the most common use-case for hiding comments, it would be nice to make it faster.

Maybe a keyboard shortcut, so you hold a certain key down while clicking the comment?

// @jamiebuilds

### PR DESCRIPTION
**Pull request by bfred-it**
From **5 clicks + 700px mouse movement** to **3 clicks + 100px movement.**

demo

Fixes #1268
Replaces and closes #1410

- [x] works on regular comments
- [x] works on inline PR comments (in Conversation or Files view)
- [x] works on ajaxed-in comments
	- [x] via `Load more...` (tested on https://github.com/facebook/react/issues/3398)
	- [x] on new comments
- [ ] ~~works when the hide button is always visible (not in a dropdown)~~ (I haven't seen this one)

### CODE REVIEW

**Comment by bfred-it** [comment]
test

**Comment by bfred-it** [comment]
test

**Comment by bfred-it** [comment]
I thought of making the secondary dropdown because I thought the Hide button would still appear outside the menu sometimes... but I can't find that anymore.

If the Hide button is always inside the dropdown, I can shorten the code by just replacing the content of the dropdown on click instead of opening a new one.

**Comment by bfred-it** [comment]
demo

File: source/content.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ import displayIssueSuggestions from './features/display-issue-suggestions';
 import addPullRequestHotkey from './features/add-pull-request-hotkey';
 import openSelectionInNewTab from './features/add-selection-in-new-tab';
 import showFollowersYouKnow from './features/show-followers-you-know';
+import hideCommentsFaster from './features/hide-comments-faster';
```
Comment (yakov116) [inline source/content.js]: It may just be me. But `quicklyhidecomments` would sound a bit better.

**Comment by bfred-it** [comment]
Should be good to go

**Pull request merged**